Source/Upstream: Yes, in v238-stable branch

From d620595dac516cce112e604a295989b18bda07fc Mon Sep 17 00:00:00 2001
From: Daniel Lin <ephemient@gmail.com>
Date: Thu, 8 Mar 2018 02:44:11 -0500
Subject: [PATCH 07/51] hwdb: add accelerometer mount matrix for Eve V (#8382)

(cherry picked from commit fc17f194ded93b51899aa0bbd4e66870d975fe5a)
---
 hwdb/60-sensor.hwdb | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/hwdb/60-sensor.hwdb b/hwdb/60-sensor.hwdb
index 77de5d21b..2c8c4467e 100644
--- a/hwdb/60-sensor.hwdb
+++ b/hwdb/60-sensor.hwdb
@@ -147,6 +147,12 @@ sensor:modalias:acpi:KIOX000A*:dmi:*:svnCube:pni16:*
 sensor:modalias:acpi:ACCE0001*:dmi:*svnEndless*:*pnELT-NL3*
  ACCEL_MOUNT_MATRIX=0, 1, 0; 0, 0, -1; -1, 0, 0
 
+#########################################
+# Eve Technology
+#########################################
+sensor:modalias:acpi:KIOX000A*:dmi:*:svnEVE*:pnEveV:*
+ ACCEL_MOUNT_MATRIX=0, 1, 0; -1, 0, 0; 0, 0, 1
+
 #########################################
 # GP-electronic
 #########################################
-- 
2.16.3

